src/main/java/lotto/domain/AutoLottoCreator.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class AutoLottoCreator implements LottoCreator {
+
+    @Override
+    public Lotteries createLotteries(int buyLottoCount) {
+        List<Lotto> lotteries = new ArrayList();
+        for (int i=0; i<buyLottoCount; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < buyLottoCount; i++) {
```
여기도 한 칸씩만 공백 넣어주세요 :)
컨벤션이기도 하고 가독성도 좋아집니다.

Reply:
지난번에 피드백주셔서 신경써서 찾아가지고 다 고쳤다고 생각했는데.. 빠진부분이 있었군요ㅠㅠ 
수정하고 앞으로 주의하겠습니다!